src/components/Frame/Frame.scss

```diff
@@ -86,15 +88,32 @@ $skip-vertical-offset: rem(10px);
     opacity: 1;
   }
 
+  &:hover {
+    background-color: var(--p-overlay);
+  }
+
   &:focus {
     border-radius: border-radius();
-    background-color: rgba(color('white'), 0.16);
+    background-color: var(--p-override-transparent, rgba(color('white'), 0.16));
```

Comment:
We don't need a background color on focus anymore. The focus ring should be enough